### DISCUSSION TITLE
Roadmap for remaining components in Material 3

### DISCUSSION THREAD

**Discussion by cmizesudo** [material-components/material-web#5386] [General]
Is there any timeframe you can give us on the completion of material-web for the remaining components in material 3.   I saw this (https://github.com/material-components/material-web/blob/main/docs/roadmap.md), but I'm wondering if you could put an expected completion timeframe on the remaining components?

Many Thanks for all your hard work!

**Comment by asyncliz** [comment]
Stay tuned! I have an updated roadmap and end of year announcements coming this week :)
